compiler/rustc_const_eval/src/interpret/memory.rs

```diff
@@ -937,8 +937,12 @@ impl<'tcx, M: Machine<'tcx>> InterpCx<'tcx, M> {
         // (both global from `alloc_map` and local from `extra_fn_ptr_map`)
         if let Some(fn_val) = self.get_fn_alloc(id) {
             let align = match fn_val {
-                FnVal::Instance(instance) => {
-                    self.tcx.codegen_instance_attrs(instance.def).alignment.unwrap_or(Align::ONE)
+                FnVal::Instance(_instance) => {
+                    // FIXME: Until we have a clear design for the effects of align(N) functions
+                    // on the address of function pointers, we don't consider the align(N)
+                    // attribute on functions in the interpreter.
+                    //self.tcx.codegen_instance_attrs(instance.def).alignment.unwrap_or(Align::ONE)
```

Comment:
```suggestion
                    // attribute on functions in the interpreter.
```
Code in comments just rots, let's get rid of it.